internal/core/command/rest.go

```diff
@@ -17,16 +17,19 @@ import (
 	"encoding/json"
 	"net/http"
 
-	mux "github.com/gorilla/mux"
+	"github.com/gorilla/mux"
+	"github.com/edgexfoundry/edgex-go/internal"
+	"runtime"
+	"fmt"
+	"github.com/edgexfoundry/edgex-go/pkg/clients"
```

Comment:
Please re-organize imports

Reply:
Re-organized imports so the start with the most high-level library (language-level), on down to low-level libraries (project-level).